app/org/sagebionetworks/bridge/play/controllers/TaskController.java

```diff
@@ -54,10 +54,11 @@ public Result getTasks(String untilString, String offset, String daysAhead) thro
         } else {
             throw new BadRequestException("Supply either 'until' parameter, or 'daysAhead' and 'offset' parameters.");
         }
-        // The ClientInfo will be added to the scheduling context.
         ClientInfo clientInfo = getClientInfoFromUserAgentHeader();
+        
         ScheduleContext context = new ScheduleContext.Builder()
             .withStudyIdentifier(session.getStudyIdentifier())
+            .withClientInfo(clientInfo)
```

Comment:
Should we add a check to TaskControllerTest to make sure we're propagating the clientInfo correctly?